source/_docs/backups.md

```diff
@@ -225,3 +225,16 @@ Additionally, you can manually trigger a full Pantheon backup job for any site e
 ```bash
 terminus backup:get <site>.<env> --file=<filename> --element=<element>
 ```
+### What Can I Do During File Backups?
+
+From the Backup Log tab on the Site Dashboard, you can see the status of currently running backups for your code, database, and files:
+
+![Backups in progress](/source/docs/assets/images/backup-progress.png)
+
+ - **Code** is anything version controlled and commited via the Site Dashboard.
+ - **Database** is a `mysqldump` of your site's db.
+ - **Files** refers to *static files*, which are anything uploaded via SFTP or your CMS that isn't version controlled. This includes image galleries.
```

Comment:
Suggest adding the path to the public files dir for additional context (e.g. `wp-content/uploads` for WordPress and `sites/default/files` for Drupal).